### PR TITLE
Add invalidateCaches option for CadenzaClient#reload

### PR DESCRIPTION
**Pull request by MrLynx93**

### CODE REVIEW

**Review by klschm — APPROVED** [review]

**Review by jkissel — CHANGES_REQUESTED** [review]

File: src/cadenza.js [hunk 1]
```diff
@@ -809,10 +809,11 @@ export class CadenzaClient {
 
   /**
    * Reload the views of a worksheet.
+   * @param {boolean} invalidateCaches - When true, caches will be invalidated for objecttypes used in the worksheet
    */
-  reload() {
-    this.#log('CadenzaClient#reload');
-    this.#postEvent('reload');
+  reload(invalidateCaches = false) {
```
[the remark this answers is not in the document]
Comment (MrLynx93) [reply src/cadenza.js]: I updated that in amended commit, you can have a look again

File: src/docs.md [hunk 2]
```diff
@@ -359,10 +359,16 @@ button.onclick = () => cadenzaClient.downloadData('{embeddingTargetId}', 'excel'
...
 <small>API: [CadenzaClient#reload](./classes/CadenzaClient.html#reload)</small>
 
+To reload a worksheet:
 ```javascript
 cadenzaClient.reload();
 ```
 
+To invalidate caches and reload:
+```javascript
+cadenzaClient.reload(true);
```
Comment (klschm) [inline src/docs.md:369]: Shouldn't here the parameter _true_ be wrapped in a JSON object?
`cadenzaClient.reload({invalidateCaches: true})`

**Review by klschm — CHANGES_REQUESTED** [review]

File: src/docs.md (hunk 2 above)
Comment (MrLynx93) [reply src/docs.md]: Right, I changed that too, thanks :+1:

**Review by klschm — APPROVED** [review]

File: src/docs.md (hunk 2 above)
Comment (klschm) [reply src/docs.md]: :+1:

**Pull request merged**